source/reference/api/start.txt

```diff
@@ -91,6 +91,25 @@ Request Body Parameters
      - Required
      - Name of the destination cluster.
 
+   * - ``buildIndexes``
+     - boolean
+     - Optional
+     - Configures index builds during sync.
+
+       Supported Options:
+
+       * ``beforeDataCopy`` (the default) causes ``mongosync`` to build indexes
+         on the destination cluster.  These include both existing indexes and
+         any indexes created during migration on the source cluster.
+
+       * ``never`` causes ``mongosync`` to skip building non-essential indexes
```

Comment:
"omit" not "skip"

Also, consider using "unnecessary" rather than "non-essential".

Reply:
Fixed